Code/Legacy/CrySystem/XConsole.cpp

```diff
@@ -452,9 +455,21 @@ void CXConsole::RegisterVar(ICVar* pCVar, ConsoleVarFunc pChangeFunc)
     }
 
     ConsoleVariablesMapItor::value_type value = ConsoleVariablesMapItor::value_type(pCVar->GetName(), pCVar);
-
     m_mapVariables.insert(value);
 
+    auto consoleInterface = AZ::Interface<AZ::IConsole>::Get();
+    if (consoleInterface != nullptr)
```

Comment:
nit: Could reduce to...

```
if (auto consoleInterface = AZ::Interface<AZ::IConsole>::Get())
{
    ...
```

This makes `consoleInterface` harder to use incorrectly too

Reply:
Good point, but I think it's considered better style to use the [if initializer](https://www.tutorialspoint.com/cplusplus17-if-statement-with-initializer) now, so I'll use that. Thanks!